lib/logstash/inputs/redis.rb

```diff
@@ -132,6 +133,7 @@ def internal_redis_builder
   def connect
     redis = new_redis_instance
     load_batch_script(redis) if batched? && is_list_type?
+    metric.increment(:connections)
```

Comment:
Not sure `connections` is the right term here, I think it might confuse users into thinking that this is the `current number of connection`, Would it be better to just track retries?